src/components/Property/Types/VectorProperty/ViewOptions/ColorVector.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, RGBA } from '@mantine/core';
+
+import { ColorPicker } from '@/components/ColorPicker/ColorPicker';
+import { VectorPropertyProps } from '@/components/Property/Types/VectorProperty/VectorProperty';
+
+import { ValueList } from './DefaultValueList';
+
+export function ColorVector(props: VectorPropertyProps) {
+  const { disabled, setPropertyValue, value, isInt } = props;
```

Comment:
```suggestion
export function ColorVector({disabled, setPropertyValue, value, isInt} : VectorPropertyProps) {
```
??

Reply:
Unfortunately, no. This property type uses both the regular view for properties and for that one I need to spread all the inputted properties. And including the entire list here seemed less readable to me

Maybe it's just a bad design of the components. Would be open to discuss alternative solutions

Reply:
Yeah let's try redesign at some point :)